src/xbps/repositories/mirrors/index.md

```diff
@@ -47,6 +47,7 @@ sub-repository.
 | <https://ftp.gnome.org/mirror/voidlinux/>          | EU: Sweden        |
 | <https://ftp.lysator.liu.se/pub/voidlinux/>        | EU: Sweden        |
 | <https://ftp.sunet.se/mirror/voidlinux/>           | EU: Sweden        |
+| <https://mirror.ps.kz/voidlinux/>                  | KZ: Kazakhstan    |
```

Comment:
You probably want to use `Asia: Kazakhstan`, instead. Unless there is a city in Kazakhstan named Kazakhstan? Even then, I would use `Asia: Kazakhstan, KZ`

Reply:
Thanks, I changed it to `Asia: Kazakhstan`, because a longer string may break the list formatting...

Reply:
If you'd like to increase the column width, I don't have any issue with it!